e2e/benchmarks/model_config.js

```diff
@@ -135,7 +135,7 @@ const benchmarks = {
     loadTflite: async (enableProfiling = false) => {
       const url =
           'https://tfhub.dev/tensorflow/lite-model/mobilenet_v2_1.0_224/1/metadata/1';
-      await tfliteModel.load(url, {enableProfiling});
+      return await tfliteWorkerAPI.loadTFLiteModel(url, {enableProfiling});
     },
     predictFunc: () => {
```

Comment:
Nit: This implementation for `MobileNetV2Lite` looks the same as the implementation for `MobilenetV3`. Should we factor them out into a shared function?

Reply:
Thanks @mattsoulanille, good point, done!